src/components/bigLogo.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import InViewMonitor from 'react-inview-monitor';
+import MtSvgLines from 'react-mt-svg-lines';
+
+import '../../node_modules/animate.css/animate.min.css';
```

Comment:
there's probably a better way to import...

Reply:
Is trying to import from cdn a good choice?

Reply:
I think the current version is the best way unless this works: https://www.npmjs.com/package/react-animated-css